internal/models/helmrepository/searchscope.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package helmrepositoryclustermodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceFluxcdHelmRepositorySearchScope Scope to search by, any fields left empty will be considered all (*).
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.fluxcd.helm.repository.SearchScope
+type VmwareTanzuManageV1alpha1ClusterNamespaceFluxcdHelmRepositorySearchScope struct {
```

Comment:
I think these are namespace level resources. please check other implementations. The package can be `helmrepositorynamespacemodel`

Reply:
so in other implementations for cluster level resources we named package `<resourcename>clustermodel`
and helm repository is cluster level resource so that's why I've taken `helmrepositoryclustermodel` as package name